Port planning_scene_interface to ROS2

anasarrak opened the pull request.

anasarrak committed:
Port planning_scene_interface to ROS2

anasarrak committed:
Apply clang-format

LanderU commented:
The error is the next one:

```bash
--- stderr: moveit_ros_planning_interface
CMake Error at CMakeLists.txt:19 (find_package):
  By not providing "Findmoveit_ros_move_group.cmake" in CMAKE_MODULE_PATH
  this project has asked CMake to find a package configuration file provided
  by "moveit_ros_move_group", but CMake did not find one.
  Could not find a package configuration file provided by
  "moveit_ros_move_group" with any of the following names:
    moveit_ros_move_groupConfig.cmake
    moveit_ros_move_group-config.cmake
  Add the installation prefix of "moveit_ros_move_group" to CMAKE_PREFIX_PATH
  or set "moveit_ros_move_group_DIR" to a directory containing one of the
  above files.  If "moveit_ros_move_group" provides a separate development
  package or SDK, be sure it has been installed.
```

anasarrak committed:
Update move_group cmakelist

anasarrak committed:
Update planning_interface package.xml

anasarrak commented:
The problem should be fixed on this commit https://github.com/AcutronicRobotics/moveit2/pull/59/commits/dcd9932527999e4df35061e01756494cdde2fc39

ahcorde committed:
resource retriever from sources

Update osx-compile-moveit2.sh

Update osx-install-ros2-binary.sh

Update moveit2.repos

Update package.xml

Update osx-compile-moveit2.sh

Commit:
Merge branch 'master' into planning_scene_interface

LanderU committed:
Restore moveit2.repos and remove it from OSX

ahcorde commented:
https://travis-ci.org/AcutronicRobotics/moveit2/builds/529294163
Linux and OSX!
